TESTING.asciidoc

```diff
@@ -312,6 +284,46 @@ You can also skip this by using the "dev" profile:
 mvn test -Pdev
 ---------------------------------------------------------------------------
 
+== Backwards Compatibility Tests
+
+Running backwards compatibility tests is disabled by default since it
+requires a release version of Elasticsearch to be present on the test system.
+Its fairly easy to fetch one though:
+
+---------------------------------------------------------------------------
+python3 dev-tools/get-bwc-version.py 2.0.0-rc1
+---------------------------------------------------------------------------
+
+Then you can run the backwards compatibility tests:
+
+---------------------------------------------------------------------------
+mvn verify -pl core -Dskip.unit.tests -Pdev -Dtests.filter="@backwards and not @awaitsfix" -Dtests.bwc.version=2.0.0-rc1 -Dtests.bwc.path="$(pwd)/backwards" -Dtests.security.manager=false -Dtests.seccomp=false -Dtests.jvms=1
+---------------------------------------------------------------------------
+
+Lets explain that huge command
+
+* `mvn verify`: The backwards compatibility tests are integration tests so they
+run at the verify phase.
+* `-pl core`: All the backwards compatibility tests are currently in the core
+submodule.
+* `-Dskip.unit.tests`: None of the backwards compatibility tests are unit tests
+so we have to skip the phase entirely or else it would fail because we fail the
+unit test phase if it runs *and* doesn't run any tests.
+* `-Pdev`: Skip some static analysis checks.
+* `-Dtests.filter="@backwards and not @awaitsfix"`: Filter the test set down to
+just the backwards compatibility tests that aren't marked as broken.
+* `-Dtests.bwc.version=2.0.0-rc1`: Specify the older version to run against.
+* `-Dtests.bwc.path="$(pwd)/backwards"`: Specify the directory where the older
+version lives.
+* `-Dtests.security.manager=false`: Turn off the security manager because it
+gets in the way of the backwards compatibility tests. We want to improve the
+tests so that this isn't required.
+* `-Dtests.seccomp=false`: Seccomp stops you from forking and the backwards
+compatiblity tests fork the backwards nodes. So we can't have this at all!
+* `-Dtests.jvms=1`: The backwards compatibility tests communicate over the
```

Comment:
I missed there was an explanation on this, sorry.  I don't remember this being required before, maybe I am wrong. Is it a new requirement due to recent changes? I thought that like with integration test, each jvm would be isolated and it would be possible to distribute tests over them (not that it's crazy important at this stage, just curious)